File: 0069A.Young.Physicist/solution.cpp
#include <iostream>


void solution() {
    int case_count;
    std::cin >> case_count;

    int x, y, z, sum_x = 0, sum_y = 0, sum_z = 0;
    for (int case_index = 0; case_index < case_count; ++case_index) {
        std::cin >> x >> y >> z;

        sum_x += x;
        sum_y += y;
        sum_z += z;
    }

    if (sum_x == 0 && sum_y == 0 && sum_z == 0) {
        std::cout << "YES" << '\n';
    } else {
        std::cout << "NO" << '\n';
    }
}


void setup() {
    std::ios_base::sync_with_stdio(false);
    std::cin.tie(NULL);
}


int main() {
    setup();
    solution();
}

File: 0069A.Young.Physicist/solution.py
def solution():
    case_count = int(input())

    sum_x, sum_y, sum_z = 0, 0, 0
    for _ in range(case_count):
        x, y, z = [int(x) for x in input().split()]
        sum_x += x
        sum_y += y
        sum_z += z

    if sum_x == 0 and sum_y == 0 and sum_z == 0:
        print("YES")
    else:
        print("NO")


if __name__ == "__main__":
    solution()
